lib/studies/reply/adaptive_nav.dart

```diff
@@ -33,12 +33,13 @@ class _AdaptiveNavState extends State<AdaptiveNav> {
   int _selectedIndex = 0;
 
   Widget _currentInbox;
+  UniqueKey _inboxKey = UniqueKey();
```

Comment:
final?

Reply:
Not final, since this key needs to be reset to transition between different inboxes.